### PR TITLE
Declaring PoCR networks

### PR DESCRIPTION
**Pull request by guenoledc**
We would like to reserve the chainID 1804 and 2606 as we are preparing the launch of a new Ethereum based network.

The testnet, Kerleano, is active, but not final https://ethereum-pocr.github.io/ 

We are planing production launch in July or September

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Please add a field status with the value `incubating` and remove the explorer as I cannot verify the EIP compatibility right now

**Comment by guenoledc** [comment]
> Please add a field status with the value `incubating` and remove the explorer as I cannot verify the EIP compatibility right now

Done.

As a clarification, could you tell what you need to do to verify the `EIP compatibility` (I guess EIP 155) in the explorer.
 If you want to verify the compatibility of the chain I can give you a little CRC in the kerleano network.

**Comment by guenoledc** [comment]
Any news on the PR please?

**Comment by guenoledc** [comment]
Hi @ligi 

Could you action this request please ?

**Comment by ligi** [comment]
can you please rebase so the actions run - then I can merge if they are OK

**Comment by guenoledc** [comment]
@ligi I synchronized my fork with the source one (I suppose this is what you expected) so this should do it.

Thanks

File: _data/chains/eip155-2606.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "PoCRNet",
+  "title": "Proof of Carbon Reduction mainnet",
+  "chain": "CRC",
+  "network": "mainnet",
```
Comment (ligi) [inline _data/chains/eip155-2606.json:5]:
```suggestion
```

File: _data/chains/eip155-1804.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Kerleano",
+  "title": "Proof of Carbon Reduction testnet",
+  "chain": "CRC",
+  "network": "testnet",
```
Comment (ligi) [inline _data/chains/eip155-1804.json:5]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]
please apply the changes and run the prettier

File: _data/chains/eip155-1804.json (hunk 2 above)
Comment (guenoledc) [reply _data/chains/eip155-1804.json]: Done

File: _data/chains/eip155-2606.json (hunk 1 above)
Comment (guenoledc) [reply _data/chains/eip155-2606.json]: Done

**Comment by guenoledc** [comment]
@ligi Any news?

We are due to go live soon and I would appreciate registering our chainId asap

**Comment by guenoledc** [comment]
@ligi could you please revert on what else is needed or please  approve the merge

**Comment by guenoledc** [comment]
@ligi, sorry to chase again, but I would really be nice if this could move forward.

If there is any issue with my request please clarify, I would gladly do the necessary corrections

**Comment by guenoledc** [comment]
@ligi Many thanks for proceeding

I can see the records in https://chainlist.network/ but not yet in https://chainlist.org/
Is there an expected delay of synchronization ?

**Comment by guenoledc** [comment]
> @ligi Many thanks for proceeding
> 
> I can see the records in https://chainlist.network/ but not yet in https://chainlist.org/ Is there an expected delay of synchronization ?

I can see that it is there now. So thanks VM.

**Pull request merged**